doc/classes/@GlobalScope.xml

```diff
@@ -2764,6 +2764,8 @@
 			[b]Note:[/b] A flag value must be at least [code]1[/code] and at most [code]2 ** 32 - 1[/code].
 			[b]Note:[/b] Unlike [constant PROPERTY_HINT_ENUM], the previous explicit value is not taken into account. For the hint [code]"A:16,B,C"[/code], A is 16, B is 2, C is 4.
 		</constant>
+		<constant name="PROPERTY_HINT_FLAG_LIST" value="38" enum="PropertyHint">
+		</constant>
```

Comment:
Needs description.

Reply:
Ah I have to remove this, I completely forgot. Thanks for the reminder.